packages/core/src/modules/vc/deriveProof.ts

```diff
@@ -51,9 +52,8 @@ export const deriveProof = async (
   if (proofs.length === 0) {
     throw new Error(`There were not any proofs provided that can be used to derive a proof with this suite.`)
```

Comment:
We should throw an AriesFramework Error or BbsSignatures Error? How do we do the error class for modules? Always AriesFramework or does the module have a specific error class?

Reply:
Create an issue for this! There's a lot of other places where we do this in the vc code, so would like to address this in a separate PR